VMS.Application/Services/ReportService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading.Tasks;
+using VMS.Application.Interfaces;
+using VMS.Application.ViewModels;
+using VMS.Domain.Interfaces;
+using VMS.Domain.Models;
+using VMS.Infrastructure.Data.Context;
+
+namespace VMS.Application.Services
+{
+    public class ReportService : BaseService, IReportService
+    {
+        public ReportService(IIdentityService identityService,
```

Comment:
identityService never used. Should we remove it?

Reply:
Make sense, fixed!